.circleci/scripts/binary_windows_build.sh

```diff
@@ -14,8 +14,10 @@ fi
 
 if [[ "$DESIRED_CUDA" == 'xpu' ]]; then
     export USE_SCCACHE=0
-    export XPU_VERSION=2025.0
+    export XPU_VERSION=2025.1
     export XPU_ENABLE_KINETO=1
+    # Need update kineto
+    export USE_KINETO=0
```

Comment:
What's the plan to update kineto and enable this back? Will that be done in time for PT 2.8?

Reply:
After the PR #152007 landed, I will revert this commit